js/app-functions/number-guess.js

```diff
@@ -0,0 +1,25 @@
+export const setUpNumberGuess = () => {
+  const numberSubmit = document.querySelector("#numberSubmit");
+  let correctNumber = Math.floor(Math.random() * 10);
+
+  numberSubmit.addEventListener("click", (event) => {
+    event.preventDefault();
+    checkGuess(correctNumber);
+  });
+};
+
+const checkGuess = (correctNumber) => {
+  const numberInput = document.querySelector("#numberInput");
+  let guess = Number(numberInput.value);
+
+  if (guess === correctNumber) {
+    guessResult.textContent = "Correct!";
+  } else {
+    guessResult.textContent = "Keep guessing!";
+  }
```

Comment:
Could do a higher/lower system here, and maybe keep track of the guesses?

Reply:
You can add styles as well, like green for correct, orange if it's close and red if it's far off